.github/workflows/ci.yml

```
@@ -29,12 +29,7 @@ jobs:
         run: vcpkg/vcpkg install @vcpkg_x86-windows.txt
 
       - name: Build application
-        uses: ashutoshvarma/action-cmake-build@master
-        with:
-          build-dir: ${{ runner.workspace }}/openspades.mk
-          build-type: RelWithDebInfo
-          configure-options: -A Win32 -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake -D VCPKG_TARGET_TRIPLET=x86-windows-static
-          parallel: 8
+        run: mkdir "${{ runner.workspace }}/openspades.mk"; cd "${{ runner.workspace }}/openspades.mk"; cmake .. -A Win32 -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake -D VCPKG_TARGET_TRIPLET=x86-windows-static
```

The source directory is located at `${{ runner.workspace }}/openspades` because of the `path` parameter of `actions/checkout@v1`.

https://github.com/yvt/openspades/actions/runs/4906775042/jobs/8761449087#step:2:13

```suggestion
        run: |
          mkdir "${{ runner.workspace }}/openspades/openspades.mk"
          cd "${{ runner.workspace }}/openspades/openspades.mk"
          cmake .. -A Win32 \
            -D CMAKE_BUILD_TYPE=RelWithDebInfo \
            -D CMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/openspades/vcpkg/scripts/buildsystems/vcpkg.cmake \
            -D VCPKG_TARGET_TRIPLET=x86-windows-static
```